internal/receiver/databricksreceiver/databricks_metrics_provider.go

```diff
@@ -0,0 +1,93 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package databricksreceiver
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+
+	"github.com/signalfx/splunk-otel-collector/internal/receiver/databricksreceiver/internal/metadata"
+)
+
+// dbrMetricsProvider wraps a databricksService and provides metrics for databricks
```

Comment:
Are the flat structure and required `dbr`, `databricks_`, `sparkDbr`, `spark_`, etc. prefixes clearer than inner ~`databricks` and `spark` packages?

Reply:
Great idea. Fixed.